features/step_definitions/lint-code.rb

```diff
@@ -0,0 +1,63 @@
+Then(/tuist lints project's code and passes/) do
+  out, _, _ = Open3.capture3("swift", "run", "tuist", "lint", "code", "--path", @dir)
```

Comment:
Defining assertions on the standard outout will make the tests a bit brittle. What I suggest instead is checking whether the test exited successfully or not:

```rb
out, err, status = Open3.capture("swift", "run", "tuist", "lint", "code", "--path", @dir)
flunk(err) unless status.success?
```

If you notice, `status.success?` returns false if the command failed. In that case, we use [flunk](https://apidock.com/ruby/MiniTest/Assertions/flunk) to fail with a message.

Reply:
In case it needs more clarification. If we check that the output is `Done linting!` and they change the output to be `Finished!` our tests will break.

Reply:
The problem I have here is `SwiftLint` returns code 0 even if finishes with violations found. 

<img width="788" alt="Zrzut ekranu 2020-08-31 o 21 19 07" src="https://user-images.githubusercontent.com/4774319/91758077-b07d9800-ebcf-11ea-8d31-efda4fced80d.png">

Reply:
Or maybe Im thinking in wrong way. 

Maybe I should not test `swiftlint` but `tuist` itself. So the validation result is not important here. Important is if `tuist` finishes with success. 

🤔